rero_ils/modules/local_fields/jsonschemas/local_fields/local_field-v0.0.1.json

```
@@ -0,0 +1,163 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Local fields",
+  "description": "Local fields",
```

```suggestion
  "description": "Local fields allows to add data that is available at the organisation level only.",
```
The description is meant to bring more information than the title.

---

done